src/View/Editor/edit.html.twig

```diff
@@ -22,16 +22,19 @@
     <label class="editor-label" for="height">Hauteur: </label>
     <input class="editor-input" type="number" name="height" id="height" value="{{ level.height }}" required>
     <div class="level-grid" style="--grid-width: {{ level.width }}; --grid-height: {{ level.height }};">
-    {% for row in grid %}
-        {% for cell in row %}
+    {% for y, row in grid %}
+        {% for x, cell in row %}
             <div class="level-tile">
-                {% set name = "cell-" ~ loop.index0 ~ "-" ~ loop.parent.loop.index0 %}
+                {% set name = "cell-" ~ x ~ "-" ~ y %}
```

Comment:
x and y are the cell position, aren't they?

Reply:
correct, the previous implementation wasn't functional if the arrays weren't 0 indexed

Reply:
then you should be able to generate x and y in the controller, and store them in `grid` cells ;)

(and get rid of the double loop :muscle:)